security/cert_manager_operator/index.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-about"]
+= {cert-manager-operator} overview
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-about
+
+toc::[]
+
+The {cert-manager-operator} is a cluster-wide service that provides application certificate lifecycle management. The {cert-manager-operator} allows you to integrate with external certificate authorities and provides certificate provisioning, renewal, and retirement.
+
+:FeatureName: The {cert-manager-operator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+// About the {cert-manager-operator}
+include::modules/cert-manager-about.adoc[leveloffset=+1]
+
+// Certificate request methods
+include::modules/cert-manager-request-methods.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="cert-manager-operator-about_additional-resources"]
+== Additional resources
+
+* link:https://cert-manager.io/docs/[Upstream cert-manager documentation]
```

Comment:
I think we could remove the "upstream" word from here. I also spotted another place in the release notes.

Reply:
@slaskawi Is there another wording that we could use to make it clear that it isn't our documentation? What about "cert-manager project documentation"? Thoughts?

Reply:
Yes, definitely better!